src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/TraceEventExtensions.cs

```diff
@@ -410,11 +451,11 @@ private static void HandleHistogram(TraceEvent obj, CounterConfiguration configu
             string meterName = (string)obj.PayloadValue(1);
             //string meterVersion = (string)obj.PayloadValue(2);
             string instrumentName = (string)obj.PayloadValue(3);
-            string unit = (string)obj.PayloadValue(4);
+            //string unit = (string)obj.PayloadValue(4);
             string tags = (string)obj.PayloadValue(5);
             string quantilesText = (string)obj.PayloadValue(6);
-            //int count - unused arg 7
-            //double sum - unused arg 8
+            int count = (int)obj.PayloadValue(7);
```

Comment:
From my initial investigation it looks like this is not an int, but a string.

Reply:
Added a version check